assets/js/blocks/cart/cart-cross-sells-product-list/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { ProductResponseItem } from '@woocommerce/type-defs/product-response';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProduct from './cart-cross-sells-product';
+
+const placeholderRows = [ ...Array( 3 ) ].map( ( _x, i ) => (
+	<CartCrossSellsProduct crossSellsProduct={ {} } key={ i } />
+) );
+
+interface CrossSellsProductListProps {
+	crossSellsProducts: ProductResponseItem[];
+	isLoading: boolean;
+	className?: string;
```

Comment:
Because of https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tsconfig.base.json#L10 we need to allow undefined here.

```suggestion
	className?: string | undefined;
```